pynest/nest/tests/test_sp/test_all.py

```diff
@@ -43,11 +69,27 @@ def suite():
     test_suite.addTest(test_synaptic_elements.suite())
     test_suite.addTest(test_disconnect.suite())
     test_suite.addTest(test_disconnect_multiple.suite())
-    test_suite.addTest(test_issue-578-sp.suite())
+    test_suite.addTest(test_enable_multithread.suite())
 
     return test_suite
 
 
+def getMPITestCommand():
+    # Open nestrc file and obtain the right mpi exec command
+    # Substitute the scriptfile with the test file and substitute the number
+    # of processes with 2 for this tests
+    path = os.path.expanduser('~/.nestrc')
+    nestrcf = open(path, "r")
+    for line in nestrcf:
+        if "(mpi" in line:
+            line = line.replace(" numproc", "2")
+            line = line.replace("cvs ( ) executable", "python")
+            line = line.replace("(", "")
+            line = line.replace(")", "")
+            print ("MPI test command: " + line)
+            nestrcf.close()
+            return line
+
```

Comment:
There's no need to parse and modify the line from `.nestrc` manually here as this is anyway a `SLI` function, which gives you the invocation line for a given number of processes, an executable and a script file. I suggest to replace the function signature by
```
def getMPITestCommand(np, script):
```
and the function body by something like this:
```
return nest.sli_func("mpirun", np, "python", script)
```

This is much safer, as it works for all definitions of `mpirun` in the `.nestrc` file.